src/prommis/uky/uky_flowsheet_ui.py

```diff
@@ -395,4 +397,17 @@ def get_diagram(build_options):
 
 def solve_flowsheet(flowsheet=None):
     """Solve a built/initialized flowsheet."""
+    # m = build()
+    # set_partition_coefficients(m)
+    #
+    # set_operating_conditions(m)
+    #
+    # scaled_model = set_scaling(m)
+    #
+    # initialize_system(scaled_model)
+    #
+    # results = solve(scaled_model)
+    #
+    # return results
+
     return solve(flowsheet)
```

Comment:
@dangunter The UI test is failing (see screenshot), and I'm not quite sure how to resolve it. The issue seems to be that the solve function in the flowsheet has been updated in this PR to unfix and fix certain variables after the solve. The test will pass if I replace L413 with the commented code, but I imagine we'd need to make use of the `flowsheet` input argument, which the commented code does not do. Any thoughts on this?